contrib/forms/assessment_intake/new.php

```diff
@@ -15,7 +15,6 @@
 formHeader("Form: assessment_intake");
 ?>
 <html><head>
-<?php html_header_show();?>
```

Comment:
Very very good just having 1 line of change in your files that you have committed. Lots of times ppl will get the file wrong, recreate it, or some how get the line numbering off so that it shows every single line in the whole file was a change. Lol these files will be entirely green and red and you'll see them from time to time on Github